apps/client/src/app/pages/portfolio/activities/import-activities-dialog/import-activities-dialog.component.ts

```diff
@@ -175,6 +175,117 @@ export class ImportActivitiesDialog implements OnDestroy {
     aStepper.reset();
   }
 
+  public onFilesDropped(files: FileList): void {
+    if (files.length === 0) {
+      return;
+    }
+
+    const droppedFile = files[0];
+
+    // Check the file type and handle it accordingly (JSON/CSV)
```

Comment:
From here, there is a lot of duplicated code. Can you create a function `handleFile(file) {}` with this stuff and reuse it for `onFilesDropped()` and `onSelectFile()`?

Reply:
Done